test/dkanextension/src/Drupal/DKANExtension/Context/LinkcheckerContext.php

```diff
@@ -0,0 +1,61 @@
+<?php
+
+// @codingStandardsIgnoreFile
+namespace Drupal\DKANExtension\Context;
+
+use Behat\Behat\Hook\Scope\BeforeFeatureScope;
+use Behat\Behat\Hook\Scope\AfterFeatureScope;
+use Drupal\DKANExtension\Hook\Scope\BeforeDKANEntityCreateScope;
+
+/**
+ * Defines application features from the specific context.
+ */
+class LinkcheckerContext extends RawDKANContext {
+
+  protected $old_global_user;
+  public static $modules_before_feature = array();
```

Comment:
Is there a particular reason for this variables to be public? It looks like we (this class) are managing them through the whole cycle of the use of this context.

Reply:
I based this code on what is being used in the workbench context, I have no reasoning beyond that.